### PR TITLE
TD-10941:[OPPO] dataX MongoDB迁移到TDengine

### PR DESCRIPTION
**Pull request by dingbo8128**

### CODE REVIEW

**Review by sangshuduo — COMMENTED** [review]
Basically, I did not read all of the code. I would suggest we use English as the first priority in the code. And use the Java's i18n mechanism to provide other language translations. Chinese is also very important but please provide both English and Chinese for the first time.

File: tdenginewriter/src/main/java/com/alibaba/datax/plugin/writer/tdenginewriter/DefaultDataHandler.java [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+            return 0;
+        }
+
+        try {
+            Connection conn = getTaosConnection(properties);
+            if (conn == null) {
+                return 0;
+            }
+            if (schemaManager.shouldGuessSchema()) {
+                LOG.info("无法从配置文件获取表结构信息，尝试从数据库获取");
```
[the remark this answers is not in the document]
Comment (dingbo8128) [reply tdenginewriter/src/main/java/com/alibaba/datax/plugin/writer/tdenginewriter/DefaultDataHandler.java:41]: good idea

**Review by sangshuduo — APPROVED** [review]
LGTM

**Review by zyyang-taosdata — APPROVED** [review]
LGTM

**Pull request merged**